### PR TITLE
Update collect-windows-custom-channel-logs.md

### PR DESCRIPTION
**Pull request by ajaiswals**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 1]
```diff
@@ -212,6 +216,67 @@ Configuration details:
...
+      exporters:
+        - sumologic
+```
+
+1. Create a file in folder `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\` with name `sysmon_windows.yaml`.
+2. Paste the above content into the file.
+3. Restart collector with following command:
+   ```bash title="Windows"
+   Restart-Service -Name OtelcolSumo
+   ```
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:259]: @ajaiswals , should these steps be numbered 4, 5, 6?

Comment (ajaiswals) [reply docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:259]: @kimsauce  - we can keep them 1,2,3. Thats how we have kept for other sections as well.

1,2,3 before this section is to give an overview what we will do and steps after config.yaml is the actual steps.

Probable we can add a heading after snippet. "Steps to Apply configuration" ?

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 2]
```diff
@@ -212,6 +216,67 @@ Configuration details:
...
 
+
+### Collect from Custom channels (Powershell, Sysmon)
+
+
+Following configuration demonstrates:
+
+1. **Collect**: Collect Sysmon logs.
+2. **Transform**: Set `_sourceCategory` field to `windows_event_log_prod_sysmon`.
+3. **Export**: Send data to authenticated Sumo Logic organization.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:227]:
```suggestion
* **Collect**: Collect Sysmon logs.
* **Transform**: Set `_sourceCategory` field to `windows_event_log_prod_sysmon`.
* **Export**: Send data to authenticated Sumo Logic organization.
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 3]
```diff
@@ -212,6 +216,67 @@ Configuration details:
...
+
+* **exporters**:
+  * `sumologic` Sends data to the registered Sumo Logic organization. This exporter is preconfigured in the `sumologic.yaml` file during installation.
+
+* **service**:
+  * `logs/custom_files:` Pipeline glues together the receivers with the processors and the exporters.
+
+
+
+
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:278]:
```suggestion
#### Configuration details

* **receivers**: `windowseventlog/sysmon/localhost/1690233479:` Collect logs from sysmon channel with name “Microsoft-Windows-Sysmon/Operational”.
* **processors**: `resource/windows_resource_attributes/localhost/1690233479` Adds the resource attribute `_sourceCategory` with value `windows_event_log_prod_sysmon`
* **exporters**: `sumologic` Sends data to the registered Sumo Logic organization. This exporter is preconfigured in the `sumologic.yaml` file during installation.
* **service**: `logs/custom_files:` Pipeline glues together the receivers with the processors and the exporters.
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 4]
```diff
@@ -212,6 +216,67 @@ Configuration details:
 * **service**:
   * `logs/custom_files:` Pipeline glues together the receivers with the processors and the exporters.
 
+
+### Collect from Custom channels (Powershell, Sysmon)
+
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:221]:
```suggestion
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
couple more comments

**Review by kimsauce — APPROVED** [review]

**Pull request merged**